docs/UserGuide.md

```diff
@@ -254,6 +254,45 @@ Here is an example with steps to follow:
 * You can only **find assignments with keywords of the same prefix**. For example, `find n/Assignment d/23-10-2020` will not work.
 </div>
 
+### Editing your assignment : `edit`
+
+Format: `edit INDEX PREFIX/EDITTED_FIELD`
+
+You can specify an assignment's index to edit its fields. These fields are the name, module code and deadline of assignment.
+
+This is the table of prefixes used:
+
+| Prefix | Syntax | Example |
+|-|-|
+| `n/` | n/NAME_OF_ASSIGNMENT | - `n/Tutorial` |
+| `mod/` | mod/MODULE_CODE | - `mod/ST2334` |
+| `d/` | d/DATE_OR_TIME_OF_ASSIGNMENT | - `d/24-10-2020 1300` |
+
```

Comment:
There seems to be something wrong with the formatting of the table :(

![image](https://user-images.githubusercontent.com/58943071/98031941-362af780-1e4e-11eb-80d3-cfae86ced5d7.png)